service/mantle/shipstation/OmsShipmentService.xml

```diff
@@ -0,0 +1,395 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <service verb="ship" noun="OrderItems">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="orderPartSeqId" required="true"/>
+            <parameter name="tryAutoPackage" type="Boolean" default="true"/>
+            <parameter name="shipDate" type="Timestamp" default="ec.user.nowTimestamp"/>
+            <parameter name="orderItemList" required="true" type="List"/>
+            <parameter name="trackingNo" required="true"/>
+            <parameter name="carrier" required="true"/>
+            <parameter name="labelCreateDate"/>
```

Comment:
I think we should keep names generic here similar to the field name patterns in Moqui. labelCreateDate can be labelDate.